coderd/coderd.go

```diff
@@ -185,6 +186,8 @@ func New(options *Options) *API {
 	if options.Authorizer == nil {
 		options.Authorizer = rbac.NewCachingAuthorizer(options.PrometheusRegistry)
 	}
+	// The below are no-ops if already wrapped.
+	options.Database = dbmetrics.New(options.Database, options.PrometheusRegistry)
```

Comment:
Note: I'm intentionally putting this before `dbauthz` as we already have metrics for the Authorizer.